src/components/ScaleControls.vue

```diff
@@ -197,4 +262,36 @@ p.warning {
   max-height: 12em;
   overflow-y: auto;
 }
+
+.ottava-label {
+  color: var(--color-accent-mute);
+}
+.ottava-value {
+  padding-top: 2px;
+  padding-bottom: 2px;
+  padding-left: 8px;
```

Comment:
These should probably be in `rem` in light of #859. I've used the wrong units on `main`, sorry about that.